www/templates/intro/instruction.html

```diff
@@ -0,0 +1,19 @@
+<ion-content class="has-footer">
+	<!-- <div class="intro-space"></div> -->
+	<div class="intro-title">
+	<center><ion-icon></center>
+	</div>
+	<!-- <div class="intro-space"></div> -->
+	<div class="intro-text">
+	
+	<div class="intro-space"></div>
+	<center><b>Thank you for installing e-mission. e-mission will collect data on your location and speed to make inferences about your trip such as mode choice. Please keep e-mission on your phone for at least two days. At the end of two days, you are requested to pull up your trips and answer questions to confirm inferences made by e-mission. :</b></center>
```

Comment:
Please add a note to keep location services turned on while the app is installed. Otherwise, we won't get any location :) The app automatically turns on GPS only when you are moving, so it won't drain your battery.